core/admin/mailu/api/common.py

```diff
@@ -20,14 +22,12 @@ def decorated_function(*args, **kwds):
         client_ip = flask.request.headers.get('X-Real-IP', flask.request.remote_addr)
         if utils.limiter.should_rate_limit_ip(client_ip):
             abort(429, 'Too many attempts from your IP (rate-limit)' )
-        if (request.args.get('api_token') == '' or
-            request.args.get('api_token') == None):
-            abort(401, 'A valid API token is expected as query string parameter')
-        if not hmac.compare_digest(request.args.get('api_token'), v1.api_token):
+        if not request.headers.get('Authorization'):
+            abort(401, 'A valid API token is expected which is provided as request header')
+        if not hmac.compare_digest(request.headers.get('Authorization'), v1.api_token):
```

Comment:
Hmm, you do need to have the scheme here. Please use "Bearer" as in RFC 6750 and suggested by @ghostwheel42

Reply:
The version of SwaggerUI included with flask restx does not support this out of box.
It does correctly use the request header Authorization, but it does not prefix the entered token value with `Bearer`. I wouldn't be surprised if this applies for many other clients as well.
Or you must manually enter as value `Bearer <token>` instead of `<token>` in SwaggerUI.

I could change the check to
```
hmac.compare_digest(request.headers.get('Authorization'), 'Bearer ' + v1.api_token):
```
But that does mean that the client must know that the token must be prepended with Bearer. 

We could check for both variants? check for
- `<token>`
- `Bearer <token>`

Reply:
In that case, please check whether there is a space and if there is ensure that the first part is "Bearer"